addons/binding/org.openhab.binding.tankerkoenig/src/main/java/org/openhab/binding/tankerkoenig/handler/WebserviceHandler.java

```diff
@@ -139,6 +139,13 @@ public void updateStationData() {
                 // if the result is not OK, no updates are done and the status of the Bridge goes to OFFLINE!
                 updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR,
                         "Empty return or no internet connection");
+                // if the Bridge goes OFFLINE, all Stations will go OFFLINE as well
+                for (Thing thing : getThing().getThings()) {
+                    StationHandler tkh = (StationHandler) thing.getHandler();
+                    tkh.updateStatus(ThingStatus.OFFLINE);
+                }
+                // and the scheduled job gets stopped!
+                pollingJob.cancel(true);
```

Comment:
Good Point!
I should only treat specific json-responses from tankerkoenig that way. If the json-field "OK" is true, the message should be treated normally. If this field is false, the polling should be stopped. 
In all other failure cases the polling should continue!
Thanks for that second opinion! I'll change that this evening.